<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PQC_Agent</class>
 <widget class="QMainWindow" name="PQC_Agent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PQC Agent</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string>Select a folder to begin scanning.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item><widget class="QPushButton" name="btnSelect"><property name="text"><string>Select Folder</string></property></widget></item>
      <item><widget class="QPushButton" name="btnScan"><property name="text"><string>Start Scan</string></property></widget></item>
      <item><widget class="QPushButton" name="btnExport"><property name="text"><string>Export Results</string></property></widget></item>
     </layout>
    </item>
    <item><widget class="QProgressBar" name="progressBar"><property name="value"><number>0</number></property></widget></item>
    <item><widget class="QPlainTextEdit" name="outputText"><property name="readOnly"><bool>true</bool></property></widget></item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
